tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -106,7 +115,16 @@ addr = "172.16.31.10:8287"
 #  - replace: use new entry to replace the existing entry
 #  - ignore: keep the existing entry, and ignore the new entry
 #  - error: report error and quit the program
-# on-duplicate = "replace"
+#on-duplicate = "replace"
+# Whether to detect and resolve duplicate records (unique key conflict) when the backend is 'local'.
```

Comment:
It seems that we need a `blank` between '#' and 'on-duplicate'

Reply:
there's no space in `dsn` and `keep-after-success` above though 🤔 

tbh it makes more sense if it is written like:

```toml
## Description of config (prefix with _two_ `#`)
## so that commenting and uncommenting through Ctrl+/ works
# config = '12345'

## Description of config 2
config-2 = '67890'
```

Reply:
Sorry about the no space mistake above. The Chinese version does not have such a problem. So let's just leave a space between them~